clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala

```diff
@@ -34,12 +34,16 @@ abstract class EnvIntegrationSpec(env: String)
     s"http://elasticsearch1.gotc-$env.broadinstitute.org:9200"
   )
 
-  override lazy implicit val bearerToken: OAuth2BearerToken =
-    OAuth2BearerToken(
-      AuthUtil
-        .getCredsFromServiceAccount(serviceAccount)
-        .getTokenValue
+  override lazy implicit val bearerToken: OAuth2BearerToken = {
```

Comment:
It occurs to me that this should really be `val credentials: HttpCredentials` throughout our code, as it's only ever used as type `HttpCredentials` and someday we may want to use a different authentication system.